### PR TITLE
world_countries_lists file location changed

### PR DESCRIPTION
**Pull request by nithinkvarrier**
world_countries_lists npm package files locations changed. Made changes in documentation and examples files

### CODE REVIEW

**Comment by helsonxiao** [comment]
Hi @nithinkvarrier , there are still some steps we need to complete.

1. Install the latest `world_countries_lists` for `example` & `example-nextjs` project. Make sure `package.json` & `yarn.lock` are updated as well. You could also try to run these two demo projects in your local machine.
2. Add some description in README because this is a tiny breaking change.

After that, I'll merge this pr and publish a new version. Thanks!

**Comment by helsonxiao** [comment]
Some quick-start tips for you.

https://github.com/boyuai/antd-country-phone-input/issues/23#issuecomment-982553257

File: example/yarn.lock [hunk 1]
```diff
@@ -11649,8 +11649,8 @@ worker-rpc@^0.1.0:
   dependencies:
     microevent.ts "~0.1.1"
 
-world_countries_lists@^2.1.0:
-  version "2.1.0"
+world_countries_lists@^2.4.0:
+  version "2.4.0"
   resolved "https://registry.yarnpkg.com/world_countries_lists/-/world_countries_lists-2.1.0.tgz#f93447bc7e40955fc9edde99351ece4ab67e4cda"
```
Comment (helsonxiao) [inline example/yarn.lock:11654]: I'm afraid this package is not updated. Try to run `yarn add world_countries_lists` **under each demo project directory**.

File: example-nextjs/yarn.lock [hunk 2]
```diff
@@ -361,7 +361,7 @@ util-deprecate@^1.0.1:
   resolved "https://registry.yarnpkg.com/util-deprecate/-/util-deprecate-1.0.2.tgz#450d4dc9fa70de732762fbd2d4a28981419a0ccf"
   integrity sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8=
 
-world_countries_lists@^2.3.0:
-  version "2.3.0"
+world_countries_lists@^2.4.0:
+  version "2.4.0"
   resolved "https://registry.yarnpkg.com/world_countries_lists/-/world_countries_lists-2.3.0.tgz#8eca03dc6b77c5314cd43e01c2ba5c1c50f3910e"
```
Comment (helsonxiao) [inline example-nextjs/yarn.lock:366]: +1

Comment (nithinkvarrier) [reply example-nextjs/yarn.lock:366]: Will do that. I'm using npm in development.

**Comment by helsonxiao** [comment]
Thanks again! I'll publish a new verision in this weekend.

**Pull request merged**